javascript/src/main/pages/History/Instructor.js

```diff
@@ -16,12 +18,30 @@ const Instructor = () => {
 
     const [courseJSON, setCourseJSON] = useState(initialCourseJSON);
 
+
+    const courseHeaders = [
+        { label: "courseId", key: "courseId" },
+        { label: "title", key: "title" },
+        { label: "units", key: "unitsFixed" }
+    ]
+
     return (
         <Jumbotron>
             <div className="text-left">
                 <h2>Search Archived Course Data from MongoDB</h2>
                 <h5>Search Instructor Through Various Quarters</h5>
                 <CourseSearchFormInstructor setCourseJSON={setCourseJSON} fetchJSON={fetchInstructorHistoryNameQtrJSON} />
+
+                <Button style={{margin: "1rem 0"}}>
+                    <CSVLink    
+                        style={{color: "white"}}
+                        headers={courseHeaders} 
+                        data={courseJSON.classes} 
+                        filename = {"CourseTable.csv"}>
+                        Download CSV
+                    </CSVLink>
+                </Button>
```

Comment:
Use the allow export prop to the BasicSearchTable so that you can download all the fields of the table. See the implementation in the search by name page (5pm-4-csvDownloadNameSearch branch if not merged yet).

Reply:
The prop may not work until my branch gets merged to main or unless you pull my branch into yours.